projectroles/tests/test_views.py

```diff
@@ -182,6 +183,81 @@ def test_render_anon(self):
         response = self.client.get(reverse('home'))
         self.assertEqual(response.status_code, 200)
 
+    def test_context_get_sidebar_icon_size(self):
+        """Test getting sidebar icon size in context"""
+        with self.login(self.user):
+            response = self.client.get(reverse('home'))
+            self.assertEqual(response.context['sidebar_icon_size'], 36)
+
+    @override_settings(
+        PROJECTROLES_SIDEBAR_ICON_SIZE=tags.SIDEBAR_ICON_MIN_SIZE - 2
```

Comment:
We should include these constants from `context_processors` and remove them from `tags`.